backend/package.json

```diff
@@ -54,6 +54,7 @@
     "redis": "^4.0.6",
     "url-parse": "^1.5.10",
     "web-push": "^3.5.0",
-    "winston": "^3.7.2"
+    "winston": "^3.7.2",
+    "node-fetch": "^2.6.7"
```

Comment:
`node-fetch` is already there, but in `devDependencies`, do we need them in both sets?

Reply:
We don't, probably an artifact from the merge of another changes. Fixed.